tests/CacheCompat/CommonCache.Test.MsalJava/pom.xml

```diff
@@ -27,7 +27,7 @@
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.8.6</version>
+            <version>[2.8.9,)</version>
```

Comment:
There are characters [ at the beginning and ,) at the end. Typos?

Reply:
No, this is a floating version. It means `any version >= 2.8.9`. Btw, nuget introduced them as well https://docs.microsoft.com/en-us/nuget/concepts/dependency-resolution#floating-versions

Reply:
These are not typos. This means the min version is 2.8.9 and max is the latest. See https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN402